.github/workflows/ci.yaml

```diff
@@ -9,6 +9,7 @@ jobs:
     timeout-minutes: 20
 
     strategy:
+      fail-fast: false
```

Comment:
Spurious change.

Reply:
ok cleaned up now.

Reply:
Yeah, I do that sometimes too in my repo :-)

Perhaps GitHub allows to put an expression there which we could use to only `fail-fast` on this repo...